domains/hassan.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "HassanCodez",
+        "email": "hassan.atef01100@gmail.com"
+    },
+    "record": {
+        "RECORDTYPE": "AAAA"
```

This is not a valid record object
```suggestion
        "AAAA": "your_IPv6"
```